lhotse/dataset/unsupervised.py

```diff
@@ -74,6 +74,41 @@ def _validate(self, cuts: CutSet) -> None:
         assert all(cut.has_recording for cut in cuts)
 
 
+class UnsupervisedMultiChanWaveformDataset(UnsupervisedDataset):
```

Comment:
```suggestion
class MultiChannelWaveformDataset(UnsupervisedDataset):
```

somehow reads better to me